.github/CODEOWNERS

```diff
@@ -210,6 +210,9 @@
 # ServiceLabel: %DocumentTranslator
 # AzureSdkOwners: @joheredi
 
+# PRLabel: %Cognitive - Translator
+sdk/translation/ai-translation-document-rest @hamshavathimunibyraiah
```

Comment:
```suggestion
sdk/translation/ai-translation-document-rest/ @hamshavathimunibyraiah
```

Reply:
Be aware this individual is not part of the team associated with that label for support.

Reply:
the linter doesn't catch this case?

Reply:
It should.  

//cc: @JimSuplizio

Reply:
I don't understand this comment. The entry you're adding is a source path/owner entry. If the owner is valid (meaning they have write permissions) that's all this is going to check.

Reply:
Are we talking about the same thing?    I was talking about the trailing slash for paths.   Wasn't that part of linting at one point, or am I mistaken?

Reply:
We cannot assume that the individuals responsible for pull requests are the same responsible for support - that's the entire reason that they're split.   The linter can't enforce this because it's not possible for it to determine what is correct.